utils/queryFormatUtils.ts

```diff
@@ -52,11 +52,15 @@ export const formatFedQueryForNextGen = (query: string): string => {
 export const convertValidateConsensusGenomeQuery = (query: string): string => {
   return (
     query
+      // Replace Fed variables.
+      .replace(
+        /query [\s\S]*?{/,
+        "query ($where: WorkflowRunWhereClause) {",
+      )
       // Remove fed prefix.
       .replace("fedWorkflowRuns", "workflowRuns")
       // Replace Fed arguments.
-      .replace(/input:.*\n/, "where: $where")
-      
+      .replace(/input:.*\n/, "where: $where){\n")      
```

Comment:
nit - this kind of assumes the rest of the line is `$input) {`, maybe `/input:[\s\S]*?\)/, "where: $where)"` would be better, to just match until the next `)`?